examples/strings/strings.bal

```diff
@@ -11,78 +11,77 @@ public function main() {
     string s2 = statement.toLower();
     io:println("ToLower: " + s2);
 
-    //This compares two strings ignoring the case. It returns 'true' if the strings are equal, or 'false' if they are not equal.
+    // Compare two strings ignoring the case.
+    // Returns 'true' if the strings are equal, or 'false' if they are not equal.
```

Comment:
This should be `true` . Use backtick instead of apostrophe. Check all the places.